build.gradle.kts

```
@@ -13,7 +13,7 @@ java {
 }
 
 dependencies {
-    implementation("com.github.Trigary:AdvancementCreator:v2.0")
+    implementation("com.frengor:ultimateadvancementapi:2.2.2")
```

this implementation acts like the non shaded version is used, but is declared as an implementation
this should be compileOnly
```suggestion
    compileOnly("com.frengor:ultimateadvancementapi:2.2.2")
```